solver/src/main/java/org/chocosolver/util/tools/RealUtils.java

```diff
@@ -729,4 +731,24 @@ public static RealInterval acos_wrt(RealInterval i, RealInterval res) {
         return new RealIntervalConstant(retInf, retSup);
     }
 
+    public static RealInterval updateBounds(RealInterval var, RealInterval newInterval) {
```

Comment:
this method already exists for `RealVar` and it not required anymore with the correction I suggested.